src/patternfly/components/Pagination/pagination.scss

```diff
@@ -48,14 +82,71 @@
   }
 
   &.pf-m-footer {
-    .pf-c-options-menu {
+    position: sticky;
+    bottom: 0;
+    justify-content: center;
+    padding-top: var(--pf-c-pagination--m-footer--PaddingTop);
+    padding-right: var(--pf-c-pagination--m-footer--PaddingRight);
+    padding-bottom: var(--pf-c-pagination--m-footer--PaddingBottom);
+    padding-left: var(--pf-c-pagination--m-footer--PaddingLeft);
+    margin-top: var(--pf-c-pagination--m-footer--MarginTop);
+    background-color: var(--pf-c-pagination--m-footer--BackgroundColor);
+    border-top: var(--pf-c-pagination--m-footer--BorderTopWidth) solid var(--pf-c-pagination--m-footer--BorderTopColor);
+
+    &.pf-m-no-sticky {
+      --pf-c-pagination--m-footer--MarginTop: 0;
+      --pf-c-pagination--m-footer--BorderTopWidth: 0;
+
+      position: relative;
+    }
+
+    .pf-c-button {
```

Comment:
It wouldn't apply styles since it's just var declarations, they're just scoped to `.pf-c-button`, but they really don't need to be. I scoped them to `.pf-m-footer` instead.